ui/app/components/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js

```diff
@@ -88,8 +88,23 @@ export default class AdvancedTabContent extends Component {
           'advanced-tab__gas-edit-row__input--error': isInError && errorType === 'error',
           'advanced-tab__gas-edit-row__input--warning': isInError && errorType === 'warning',
         })}>
-          <div className="advanced-tab__gas-edit-row__input-arrows__i-wrap" onClick={() => onChange(value + 1)}><i className="fa fa-sm fa-angle-up" /></div>
-          <div className="advanced-tab__gas-edit-row__input-arrows__i-wrap" onClick={() => onChange(value - 1)}><i className="fa fa-sm fa-angle-down" /></div>
+          <div
+            className="advanced-tab__gas-edit-row__input-arrows__i-wrap"
+            onClick={() => onChange(value + 1)}
+          >
+            <i className="fa fa-sm fa-angle-up" />
+          </div>
+          <div
+            className="advanced-tab__gas-edit-row__input-arrows__i-wrap"
+            onClick={() => {
+              const newValue = value >= 1
+                ? value - 1
+                : 0
+              onChange(newValue)
```

Comment:
Could we instead write `onChange(Math.max(value - 1, 0))`?

Reply:
1296bcc00